coursework/product-design/index.md

```diff
@@ -197,111 +213,116 @@ Now it's time to put your draft design to the test! The test we are conducting i
 
 ### Playing the Wizard of Oz
 
-You may find it easier go out in pairs to conduct this form of testing. One of you need to act as the computer, so you need a second person to record your results. Test with only one user at a time, not a group. This is to avoid one user’s actions and opinions from influencing the results of other users.
+You may find it easier conduct this form of testing in pairs. One of you will need to act as the computer, so you need a second person to record your results. Test with only one user at a time, not a group. This is to avoid one user’s actions and opinions from influencing the results of other users.
+
+Ask them for five to ten minutes of their time, and if they agree, start by giving them a little context for what you are about to show them. For instance, if the app in question is NUSMods, you might tell them that they have just received a link from a friend telling them to check out a new app for organizing their NUS timetable. You may also give them an explicit goal to accomplish, such as "you would like to sign up for an account". They will need to achieve this goal by interacting with your wireframes. If you are conducting this virtually, the person acting as the computer can share their screen.
 
-Ask them for five to ten minutes of their time, and if they agree, start by giving them a little context for what you are about to show them. For instance, if the app in question is NUSMods, you might tell them that they have just received a link from a friend telling them to check out a new app for organizing their NUS timetable. You may also give them an explicit goal to accomplish, such as "you would like to sign up for an account".
+Instruct them to *think aloud* and use their finger to point at where they will click or tap, or if you are doing this virtually, ask them to say out loud what they want to do. A quick way to explain “Think Aloud” to your users is “Vocalize your thoughts. Let us know what you’re thinking throughout the way”. Explain that you will be acting the part of the computer, and reassure them that there is no wrong answer. Do also tell the user to notify you if they are done using the app / website or do not have anything else to do. This will make it clear to you when the user has totally lost interest, given up on your UI, or has no more use for your website / app.
```

Comment:
Is this accurate for online user testing? Would love to find out from the TAs who took CS3216 last year.

Reply:
From my personal experience last year, it depends on the group's target demographic and familiarity with the users being interviewed. I think there's a risk of giving too much contextual knowledge to the user as he/she is navigating through the mock-up, especially when interviewing friends. The prompt should be sufficient for helping users to vocalise their thoughts but students should be aware of how much information they give away when responding to questions.

One strategy which I found effective for getting more out of quiet interviewees was writing down questions which came into my mind at certain junctures but did not ask to avoid giving away contextual information. It helps the interviewer remember to revisit specific parts of the mock-up later on while also not running the risk of giving away too much, which reduces the user's immersion into the process.

Reply:
For virtual testing, I think it might be hard for testers to vocally explain what they want to click or tap. What my group did last year was, over the Zoom meeting, we shared screen with the testers showing our Figma prototype, and we asked them to use the Zoom annotation/stamp to indicate where they wanna click. I think it is quite intuitive this way. If the tester attempts to tap at places that they are not supposed to, it is quite clear to see. However, as Kevin mentioned, this method may only work for certain demographics that are more tech-savvy.

Reply:
Thanks for the feedback @lyskevin and @eksinyue! I've updated the writeup accordingly.